ui/src/app/applications/components/filter/filter.scss

```diff
@@ -0,0 +1,56 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+.filter {
+    background-color: white;
+    padding: 10px;
+    box-shadow: 1px 1px 2px 0px rgba(0, 0, 0, 0.1);
+    border-radius: 5px;
+    margin-bottom: 1em;
+
+    &__header {
```

Comment:
I think full width input looks better:

![image](https://user-images.githubusercontent.com/426437/116454690-1c72d780-a815-11eb-87ed-34e73fa4f330.png)

instead of

![image](https://user-images.githubusercontent.com/426437/116454734-2ac0f380-a815-11eb-9cf3-3c6ae13a8a1e.png)


Can you change style so that input take full width?

Reply:
That's interesting - in Safari, it is full width. I'll make sure it's full width in all browsers

Reply:
Safary becoming new Internet Explorer :(